jihyeon/routes/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const router = express.Router();
+
+const userRouter = require("./userRoute");
+const postRouter = require("./postRoute");
+const likesRouter = require("./likesRoute");
```

Comment:
```suggestion
const express = require("express");

const userRouter = require("./userRoute");
const postRouter = require("./postRoute");
const likesRouter = require("./likesRoute");

const router = express.Router();
```
필요한 모듈들을 먼저 불러오는 순서대로 작성되는것이 컨벤션입니다!